src/qt/walletmodel.cpp

```diff
@@ -322,6 +336,26 @@ void WalletModel::setWalletCustomFee(bool fUseCustomFee, const CAmount& nFee)
     }
 }
 
+void WalletModel::setWalletStakeSplitThreshold(const CAmount nStakeSplitThreshold)
+{
+    LOCK(wallet->cs_wallet);
+    if (wallet->nStakeSplitThreshold != nStakeSplitThreshold) {
+        wallet->nStakeSplitThreshold = nStakeSplitThreshold;
+        CWalletDB(wallet->GetDBHandle()).WriteStakeSplitThreshold(nStakeSplitThreshold);
```

Comment:
now that we are here, would be good to encapsulate the `nStakeSplitThreshold` field access and the wallet db write access.

Reply:
yep, done.